src/setup-dotnet.ts

```diff
@@ -90,9 +93,16 @@ export async function run() {
       }
     );
 
-    core.setOutput('dotnet-version', versionToOutput);
+    core.setOutput(Outputs.DotnetVersion, versionToOutput);
 
-    const matchersPath = path.join(__dirname, '..', '.github');
+    if (core.getBooleanInput('cache') && isCacheFeatureAvailable()) {
+      const cacheDependencyPath = core.getInput('cache-dependency-path');
+      await restoreCache(cacheDependencyPath);
+    } else {
+      core.setOutput(Outputs.CacheHit, false);
+    }
```

Comment:
Do you think this else condition to set the `cache-hit` output is needed? I believe that the `cache-hit` output is useful when the action tried to restore cache and found that the same record exists/doesn't exist in the cache storage. The customer is interested what happened inside the restoring process. If the action didn't try to restore cache, I think we don't need to set `cache-hit` to false.